samples/boards/arduino_101/environmental_sensing/README.rst

```diff
@@ -106,17 +106,15 @@ Before using the environmental sensing sample, the Arduino 101 board needs to be
 flashed with the firmware for its nRF51 Bluetooth LE controller. To do this,
 follow the steps in :ref:`bluetooth_firmware_arduino_101`.
 
-The sample provide a single :file:`Makefile` that builds both images, simple run
-make in the top level directory of the application and flash either using JTAG
-or using DFU. (DFU is preferred).
-
-The sample can be found here: :file:`samples/boards/arduino_101/environmental_sensing` in the
-Zephyr project tree.
-
-.. code-block:: console
-
-   $ make
-   $ make flash # for flashing with JTAG
+The sample provide a single :file:`CMakeLists.txt` that builds both images,
```

Comment:
@nashif @SebastianBoe is this correct?

Reply:
no

Reply:
Fix doc or CMakeLists then?

Reply:
I suggest we fix doc. And then use this as an example of multi-isa builds when we support this in the next release.

Docs can say something like:

Build and flash both the "sensor" application and the "ap" application in sequence.

Reply:
Ah, didn't see this was resolved :)